### PR TITLE
JDK-8195669 Cross Compile Arm Swing no longer fails

### PR DESCRIPTION
**Pull request by dellgreen**

### CODE REVIEW

**Comment by dellgreen** [comment]
OCA has now been signed and excepted for Tyba Design Limited

**Comment by johanvos** [comment]
Looks ok to me.
Maybe the preceding comment in build.gradle can now be removed as well:
```
    // Now that JMX is gone, we can likely uncomment the following as
    // a fix for JDK-8195669
```
Or do we want to keep that there for historical reasons? @kevinrushforth

**Comment by kevinrushforth** [comment]
I would expect that comment should be removed, since its only purpose was to hint at the change that is now being made.

File: build.gradle [hunk 1]
```diff
@@ -2158,13 +2158,11 @@ project(":controls") {
 }
 
 project(":swing") {
-    // Now that JMX is gone, we can likely uncomment the following as
```
Comment (johanvos) [inline build.gradle]: can you remove this comment?

**Comment by johanvos** [comment]
also, can you rebase the PR to the current develop branch?

**Comment by dellgreen** [comment]
thanks for doing this, I was flat out and didn't get a chance to follow up on your requests.

**Pull request merged**